.eslintrc

```diff
@@ -22,8 +21,17 @@
                 ]
             }
         ],
+        "indent": [
```

Comment:
I will create a NPM package for our lint configs. The copy paste is getting old.